src/onnx/parse_split.cpp

```diff
@@ -74,6 +75,23 @@ struct parse_split : op_parser<parse_split>
             vec_splits.resize(info.num_outputs, dl);
         }
 
+        if(std::accumulate(vec_splits.begin(), vec_splits.end(), int64_t(0)) !=
+           static_cast<int64_t>(lens[tuned_axis]))
+        {
+            std::string output;
+            for(const auto& index : vec_splits)
```

Comment:
can you include stringutils.hpp and try `to_string_range(vec_splits)` instead of the for loop?

Reply:
On it

Reply:
Done